libraries/core/src/main/kotlin/com/bumble/appyx/core/navigation/model/combined/CombinedNavModel.kt

```diff
@@ -18,6 +19,13 @@ class CombinedNavModel<NavTarget>(
     val navModels: List<NavModel<NavTarget, *>>,
 ) : NavModel<NavTarget, Any?>, Destroyable {
 
+    init {
+        val permanentNavModelCount = navModels.filterIsInstance<PermanentNavModel<*>>().count()
```

Comment:
Slightly more efficient (as it doesn't create, then discard a list):
`val permanentNavModelCount = navModels.count { it is PermanentNavModel<*> }`